lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1892,21 +1893,22 @@ def increment_failed_requests_count(code)
     # ingested by the Stackdriver Logging API.
     def increment_ingested_entries_count(count)
       return unless @ingested_entries_count
-      @ingested_entries_count.increment({}, count)
+      @ingested_entries_count.increment({ grpc: @use_grpc, code: @ok_code },
```

Comment:
There's no point in adding a code to `ingested_entries_count` — it's always the same.

Reply:
I was thinking about combining `stackdriver_successful_requests_count` and stackdriver_failed_requests_count`. Also `stackdriver_ingested_entries_count` and `stackdriver_dropped_entries_count`.

For instance, if we look at how these metrics get tranformed (https://github.com/Stackdriver/collectd/pull/125/files). They are combinable in some sense. WDYT?

Reply:
I would propose keeping them separate in the agents and combining (if necessary) in the API-side mappings.
I'll comment on that PR separately.

Reply:
Sure. I've reverted the commit to add code explicitly. The only reason I did it is because the code is different between gRPC and REST. The former is 0, while the latter is 200. This this can be inferred via `use_grpc`.